app/views/concerns/theme_filterable_view.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+##
+# Filtering by theme
+module ThemeFilterableView
+  extend ActiveSupport::Concern
+
+  protected
+
+  def theme_filter_options
+    theme_options = theme_filters.map { |key, data| { label: data[:label], value: key } }.tap do |options|
```

Comment:
is the reason not to use 'pro_json_api_theme_filters' here, to keep it more generic so we can reuse this in for example galleries or somewehre?